src/main/java/seedu/address/logic/commands/SortByDateCommand.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.logic.commands;
+
+import java.util.Comparator;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Sorts all Persons by date
+ */
+public class SortByDateCommand extends Command {
+
+    public static final String MESSAGE_SUCCESS = "Persons list has been sorted by date.";
+
+    public static final String COMMAND_WORD = "sort_date";
+    public static final String ALT_COMMAND_WORD = "sd";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts all Persons by date recent to distant "
+            + "or distant to recent based on the paramater.\n"
+            + "Parameters: recent/distant\n"
```

Comment:
Hi @YINZE66 I think using asc and desc would be more intuitive for the users and shorter as well, what do you think?

Reply:
yeah, I think can use this, can also ask others' opinions.

Reply:
yeah, asc and desc for ascending and descending date sounds good to me